agent/api/container/container.go

```diff
@@ -78,6 +80,10 @@ const (
 
 	// neuronVisibleDevicesEnvVar is the env which indicates that the container wants to use inferentia devices.
 	neuronVisibleDevicesEnvVar = "AWS_NEURON_VISIBLE_DEVICES"
+
+	// DockerContainerMinimumMemoryInBytes is the minimum amount of
+	// memory to be allocated to a docker container
+	DockerContainerMinimumMemoryInBytes = 256 * 1024 * 1024 // 256MB
```

Comment:
This constant was different between unix (4mb) and windows (256mb). Now you've changed it to always be the windows value.

The unix value for this needs to stay at 4mb.

Reply:
added files with appropriate values